eclipse-theia/theia-blueprint
# blueprint: upgrade to `1.26.0`

#### What it does


The commit upgrades the `@theia` version of dependencies to `v1.26.0`. The change also adds the companion extension under the plugin exclusion list since it does not exist on open-vsx.

#### How to test


1. confirm that blueprint builds.
2. confirm that blueprint starts.
3. the `download:plugins` script should not fail with `ms-vscode.js-debug-companion` not being found.

#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

Signed-off-by: vince-fugnitto <vincent.fugnitto@ericsson.com>
